src/ngraph/runtime/gpu/gpu_kernel_emitters.hpp

```diff
@@ -26,6 +28,131 @@ namespace ngraph
         {
             namespace kernel
             {
+                void emit_prologue(codegen::CodeWriter& writer, const Node* node)
+                {
+                    writer << "{   //" << node->get_name() << "\n";
+                    writer.indent++;
+                }
+
+                void emit_epilogue(codegen::CodeWriter& writer)
+                {
+                    writer.indent--;
+                    writer << "}\n";
+                }
+
+                void emit_memset(codegen::CodeWriter& writer,
```

Comment:
Source belongs in cpp file, not header.

Reply:
Since these functions are serving only to modify the codegen::CodeWriter, my hope was for them to be inlined, which is why I left them in the header. Happy to move them to the cpp file if this is a design choice. Performance probably is not critical at this level.

Reply:
Implementations are now moved to the cpp file.